doc/content/gateways/thethingsindoorgateway/_index.md

```diff
@@ -24,16 +24,18 @@ The gateway EUI can later be found at the bottom in the WiFi setup screen.
 
 3. The WiFi password of the {{% ttig %}}, typically printed on the back of the gateway. 
 
+{{< note >}} Keep in mind that currently it is not possible to connect {{% ttig %}} to {{% tts %}} distributions other than {{% tts %}} Cloud and {{% tts %}} Community Edition. {{</ note >}}
```

Comment:
Out of curiosity: what are we missing from this @KrishnaIyer ?

Should we use the distributions layout for this instead, or do you think a warning is appropriate in this case?

Reply:
This is actually possible. The gateway has to first be claimed in TTS (Cloud/CE) and then can be configured to any private network. @nejraselimovic can you create an issue for me to document this process? I'll pick that up in the next milestone.